index.ts

```diff
@@ -36,11 +36,11 @@ function parseProjectName(repoName: string): string {
         .replace("/", "$");
 }
 
-function runElmApp(viewHash: string, dirPath: string, model: any): Promise<string> {
+function runElmApp(moduleHash: string, viewHash: string, dirPath: string, model: any): Promise<string> {
 
     return new Promise((resolve, reject) => {
         const Elm = require(path.join(dirPath, "elm.js"));
-        const privateName = `PrivateMain${viewHash}`;
+        const privateName = `PrivateMain${moduleHash}`;

```

Comment:
💭 why this change?

Reply:
we're combining multiple view functions into one file, so just picking the first view function to generate the hash seems arbitrary. The name would also clash if the same view function is used elsewhere.

Reply:
Gotcha